app/(app)/offence/index.tsx

```diff
@@ -79,13 +94,17 @@ const OffencePage = () => {
             <Field label={t('offence.vehicle')}>
               <TextInput
                 value={ecv}
-                className="font-source-500medium"
+                autoCapitalize="characters"
+                className="font-belfast-700bold text-[18px] color-black"
                 isDisabled={!!role?.actions.scanCheck}
                 onChangeText={(value) => setOffenceState({ ecv: value })}
```

Comment:
Shouldn't we use the same formatting function as in MPA?
trim, toUpperCase, replace whitespaces?